blinkit-app/src/homePage/molecules/header/Header.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from "react";
+import InsertInput from "../../atoms/insertInput/Insertinput"
+import LoginCart from "../loginCart/LoginCart";
+import LogoLocation from "../logoLocation/LogoLocation";
+import "./header.css";
+
+class Header extends Component {
+  render() {
+    return (
+      <section className="header">
+        <LogoLocation />
+        <div className="search">
+          <InsertInput
+            type="text"
+            placeholder="Search.."
+            classname="search-bar"
```

Comment:
We name classNames in camelCase. Second searchInput would rather be a more relevant classname